CreateItem/CreateItemWrapper.js

```diff
@@ -1,20 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Button, Icon, omitProps } from '@folio/stripes/components';
 import className from 'classnames';
 
+import { Button, Icon, omitProps } from '@folio/stripes/components';
+
 import css from './CreateItemWrapper.css';
 import CreateItemModal from './CreateItemModal';
 
-export default class CreateItemWrapper extends React.Component {
+class CreateItemWrapper extends React.Component {
```

Comment:
what do you think of using function here instead of class?

Reply:
also `dataKey: PropTypes.string,` is not used anymore. I'm not sure if that could lead to troubles

Reply:
> what do you think of using function here instead of class?

💯 Take a look, please